aimd/job/torch_mnist_trainingjob_aimd.py

```diff
@@ -0,0 +1,231 @@
+import argparse
+import logging
+import os
+import shutil
+
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn.parallel import DistributedDataParallel as DDP
+import torch.optim as optim
+from torch.utils.tensorboard import SummaryWriter
+from torchvision import datasets, transforms
+
+parser = argparse.ArgumentParser(
+    description='Distributed training of Keras model for MNIST with DDP.')
+parser.add_argument('--api_key',
+                    type=str,
+                    help='API Key for requesting AIMD server.')
+parser.add_argument(
+    '--backend',
+    type=str,
+    help='Distributed backend',
+    choices=[dist.Backend.GLOO, dist.Backend.NCCL, dist.Backend.MPI],
+    default=dist.Backend.GLOO)
+parser.add_argument('--host', type=str, help='URL of AIMD server.')
```

Comment:
use a more specific argument name. for example, `--aimd_server`

Reply:
changed to `--aimd_host`